Gruvo/Gruvo/Controllers/AuthController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Gruvo.Models;
+using Microsoft.AspNetCore.Mvc;
+using Gruvo.DAL;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authorization;
+using Gruvo.Data;
+
+namespace Gruvo.Controllers
+{
+    [Route("api/[controller]")]
+    public class AuthController : ControllerBase
+    {
+        private static MSSQL DataBase = new MSSQL("Data Source=INTEL;Initial Catalog=Gruvo;Integrated Security=True");
+
+        public static List<UserInfo> Users = DataBase.UserDAO.GetUsers().ToList();
```

Comment:
No dao call in static field initialization. It will fall and exception will not be caught.